css/app.css

```diff
@@ -1,3 +1,913 @@
+.latest-articles-item:nth-child(5) .main-article-showcase-header {
+  background:
+    linear-gradient(
+      180deg,
+      rgba(0, 0, 0, 0) 0%,
+      rgba(0, 0, 0, 0.44021358543417366) 29%,
+      rgba(0, 0, 0, 0.5634628851540616) 68%,
+      rgba(0, 0, 0, 1) 100%
+    ),
+    url(../img/latest-articles-images/latest-articles-image-5.png);
+}
+
+.latest-articles-item:nth-child(6) .main-article-showcase-header {
+  background:
+    linear-gradient(
+      180deg,
+      rgba(0, 0, 0, 0) 0%,
+      rgba(0, 0, 0, 0.44021358543417366) 29%,
+      rgba(0, 0, 0, 0.5634628851540616) 68%,
+      rgba(0, 0, 0, 1) 100%
+    ),
+    url(../img/latest-articles-images/latest-articles-image-6.jpg);
+}
+
+.latest-articles-item:nth-child(7) .main-article-showcase-header {
+  background:
+    linear-gradient(
+      180deg,
+      rgba(0, 0, 0, 0) 0%,
+      rgba(0, 0, 0, 0.44021358543417366) 29%,
+      rgba(0, 0, 0, 0.5634628851540616) 68%,
+      rgba(0, 0, 0, 1) 100%
+    ),
+    url(../img/latest-articles-images/latest-articles-image-7.gif);
+}
+
+.latest-articles-item:nth-child(8) .main-article-showcase-header {
+  background:
+    linear-gradient(
+      180deg,
+      rgba(0, 0, 0, 0) 0%,
+      rgba(0, 0, 0, 0.44021358543417366) 29%,
+      rgba(0, 0, 0, 0.5634628851540616) 68%,
+      rgba(0, 0, 0, 1) 100%
+    ),
+    url(../img/latest-articles-images/latest-articles-image-8.png);
+}
+
+.latest-articles-item:nth-child(9) .main-article-showcase-header {
+  background:
+    linear-gradient(
+      180deg,
+      rgba(0, 0, 0, 0) 0%,
+      rgba(0, 0, 0, 0.44021358543417366) 29%,
+      rgba(0, 0, 0, 0.5634628851540616) 68%,
+      rgba(0, 0, 0, 1) 100%
+    ),
+    url(../img/latest-articles-images/latest-articles-image-9.jpg);
+}
+
+.footer-social-icons li:not(:first-child),
+.footer-links li:not(:first-child) {
+  margin-left: 0.75rem;
+}
+
+/* media queries  */
+
+/* 1247 */
+
+@media (max-width: 1247px) {
+  main {
+    padding: 40px 20px;
+  }
+}
+
+/* 1025px */
+
+@media (max-width: 1023px) {
+  .nav-container {
+    display: flex;
+  }
+
+  .navbar-container > *:not(:first-child) {
+    margin-left: 105px;
+  }
+
+  .nav-top {
+    display: flex;
+  }
+
+  .nav-logo {
+    position: absolute;
+    z-index: 0;
+    width: 105px;
+    height: 80px;
+    display: flex;
+  }
+
+  .latest-news-grid-container,
+  .latest-funding-grid-container,
+  .latest-deals-grid-container {
+    grid-template-columns: 1fr 1fr 1fr;
+  }
+
+  .latest-funding-grid-container article:last-child {
+    display: none;
+  }
+
+  .latest-funding-top {
+    flex-direction: column;
+  }
+
+  .latest-funding-top h2 {
+    padding: 0;
+    margin: 0;
+  }
+
+  .latest-funding-top-right {
+    margin-bottom: 20px;
+  }
+
+  .latest-funding-top-right li:first-child a {
+    padding-left: 0;
+  }
+
+  .latest-articles {
+    grid-template-columns: 1fr 1fr;
+  }
+
+  .main-article-showcase-header {
+    height: 22.5rem;
+  }
+
+  .showcase-item-header {
+    display: none;
+  }
```

Comment:
Same style definition of the selector on line 713, group selectors with the same style definition, or create a new class.